src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,22 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.3.0
+
+### New Feature
+
+- Git Catalog to provide predefined development environments to every developer
+- Add Git Catalog tab to Okteto UI deploy
+- Remove CLI tab from Okteto UI deploy
```

Comment:
@rlamana @omnomagonz Could we consolidate these to one entry that captures the gist of the new feature? Don't think we need this itemized breakdown, do we?

E.g. 
```suggestion
- Git Catalog, enabling predefined development environments
```

Also, do we have plans to add new documentation for how-to use enable and use the feature?

Reply:
Doh, I obviously missed the other already existing thread 🤦 I guess I missed it because it was now "outdated"…
Resolving this thread, let's continue on the other one: https://github.com/okteto/docs/pull/265#discussion_r1051033934